docs: Documentation for Databricks driver (#2515)
cube-js/cube.js: docs/content/Configuration/Connecting-to-the-Database.md

[restricted][link-clickhouse-readonly] to read-only, by setting
`CUBEJS_DB_CLICKHOUSE_READONLY` to `true`.

### Databricks JDBC

> Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's on top of a JDBC driver from DataBricks, which required

Reviewer: ```suggestion
Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's based on the JDBC driver from DataBricks, which requires

```

> Starting from `v0.26.83` Cube.js provides a driver from Databricks. It's based on top of a JDBC driver from DataBricks, which required

Reviewer 2: ```suggestion
Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's on top of a JDBC driver from DataBricks, which required
```
